src/portable/st/synopsys/dcd_synopsys.c

```diff
@@ -139,7 +145,15 @@ typedef struct {
 
 typedef volatile uint32_t * usb_fifo_t;
 
+#if TUD_OPT_HIGH_SPEED
```

Comment:
We should use the `TUD_OPT_RHPORT` (0 for FS, 1 for HS) since it is perfectly fine for BSP that use OTG_HS (port1) with just built-in PHY (without external HS PHY). i.e the `CFG_TUSB_RHPORT1_MODE = OPT_MODE_DEVICE`

Reply:
Done

Reply:
PS: I see, there is a push-force, which probably cause this "revert" :)

Reply:
Thanks for catching. For some time I thought the STM32F723 OTG_HS has also a built in FS phy and that was a remainder of my test to abable the FS phy. Obvious to no avail. I reverted again. Sorry for not commenting on that above. As long as the branch is not merged, I rebase.

B.t.w, what does TUD_OPT_RHPORT stand for " Tinyusb Device Option ... ? Root Hub?

Reply:
> Thanks for catching. For some time I thought the STM32F723 OTG_HS has also a built in FS phy and that was a remainder of my test to abable the FS phy. Obvious to no avail. I reverted again. Sorry for not commenting on that above. As long as the branch is not merged, I rebase.

No problems at all

> B.t.w, what does TUD_OPT_RHPORT stand for " Tinyusb Device Option ... ? Root Hub?

Yeah that is the name, not a very good one though, I were having trouble with naming :( . the TUD_OPT_RHPORT is derived from `CFG_TUSB_RHPORTx_MODE` telling the stack which port number that is configured as device. 
https://github.com/hathach/tinyusb/blob/master/src/tusb_option.h#L149

There is a counter `TUH_OPT_RHPORT` is for host port as well, e.g F723 can have OTG HS as device and OTG FS as host. ( though host stack doesn't support STM just yet).